templates/wiki/base.html

```diff
@@ -201,7 +201,7 @@
                     <p>
                         This website uses <a href="https://python.org">Python</a> and
                         <a href="http://flask.pocoo.org/">Flask</a>, and was developed collaboratively on
-                        <a href="https://gitlab.com/python-discord/projects/site">GitLab</a>.
+                        <a href="https://github.com/python-discord/projects/site">GitLab</a>.
```

Comment:
This URL is incorrect, not only does it point to a projects group on GitHub but it still reads GitLab.

Reply:
```suggestion
                        <a href="https://github.com/python-discord/site">GitHub</a>.
```